packages/squiggle-lang/src/fr/dist.ts

```diff
@@ -212,6 +214,40 @@ export const library: FRFunction[] = [
       ),
     ],
   }),
+  maker.make({
+    name: "metalog",
+    examples: [
+      `metalog([5, 2, 2])`,
+      `metalog([{x: -2, q: 0.1}, {x: -1, q: 0.3}, {x: 0, q: 0.9}])`,
+      `metalog([{x: -2, q: 0.1}, {x: -1, q: 0.3}, {x: 0, q: 0.9}], 2)`,
+    ],
+    definitions: [
+      makeDefinition<number[]>("metalog", [frArray(frNumber)], ([a]) =>
+        Result.errMap(
+          Result.fmap(SymbolicDist.Metalog.make({ a }), vDist),
+          (x) => REOperationError(new OtherOperationError(x))
+        )
+      ),
+      makeDefinition<{ x: number; q: number }[]>(
+        "metalog",
+        [frArray(frRecord(["x", frNumber], ["q", frNumber]))],
+        ([points]) =>
+          Result.errMap(
+            Result.fmap(SymbolicDist.Metalog.fitFromCdf(points), vDist),
+            (x) => REOperationError(new OtherOperationError(x))
+          )
+      ),
+      makeDefinition<{ x: number; q: number }[], number>(
+        "metalog",
+        [frArray(frRecord(["x", frNumber], ["q", frNumber])), frNumber],
```

Comment:
Should this be `metalog([...], { terms: N })` instead of `metalog([...], N)`? Since we might want to support other parameters there in the future, e.g. `{ exact: true }`.